tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -335,3 +335,24 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: try to overwrite a file with the virus content in a public link share
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello" to "test.txt"
+    And user "Alice" has created a public link share with settings
+      | path        | /test.txt  |
+      | name        | sharedlink |
+      | permissions | change     |
+    When user "Alice" uploads file with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" to "test.txt" using the WebDAV API
```

Comment:
Can we change this step so that it's the public that overwrites the file, not the user themselves?

Reply:
We can probably use this step, please check
```feature
When the public overwrites file "test.txt" with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" using the new WebDAV API
```